### DISCUSSION TITLE
Proposed: regen vs braking

### DISCUSSION THREAD

**Discussion by MarcFinns** [teslamate-org/teslamate#1546] [Ideas]
This is a proposed feature, not sure how to do it apart from opening a bug...

The idea came after a heated conversation with some naysayers about the fact that with a Tesla brakes are used hardly ever. I was wondering whether it is possible to display the use of brakes vs regen. It might not be available from APIs, but perhaps it can be inferred?

**Describe the bug**

**Expected behavior**

**How to reproduce it (as minimally and precisely as possible)**:

**Relevant entries from the logs**

```
paste logs here
```

**Screenshots**

**Data**

**Environment**

- TeslaMate version:
- Type of installation: Docker or Manual
- OS TeslaMate is installed on:
- User OS & Browser:
- Others:

**Comment by adriankumpf** [comment]
Feature suggestions belong here: [💡 Ideas](https://github.com/adriankumpf/teslamate/discussions/categories/ideas). I'll move it

**Reply by SteveGang1** [reply]
Hi I know that you have closed this issue, but I can't find information about how much is regenerative braking on my tesla 75d?
I want to check the information in teslamate. Where can I find the information?

TeslaMate version: [2022.12.3.2]
Type of installation: Docker 
OS TeslaMate is installed on: Ubuntu
User OS & Browser: Win10
Others:

**Reply by cwanja** [reply]
It is not actually closed, it is just "changed" to an Idea instead of an Issue. The request is still open. 

Per drive, you can look at the graph where the "Power" is below zero. I do not know of a way currently to "add it" per drive yet. Away from my machine at the moment to validate some of the updated screenshots, but here is one from the main screenshot page.
https://github.com/adriankumpf/teslamate/blob/master/website/static/screenshots/drive.png

**Comment by juneidy** [comment]
Also, something similar to this, would be nice if we can see how much power we saved by regenerative braking.

Something similar to charges dashboard that says

Energy added from regen: 2kWh

Bonus if we can add the column in Drives dashboard that says kWh regenerated
